onadata/apps/main/urls.py

```diff
@@ -350,11 +350,21 @@
         XFormListViewSet.as_view({"get": "list", "head": "list"}),
         name="form-list",
     ),
+    re_path(
+        r"^projects/(?P<project_pk>\d+)/formList$",
+        XFormListViewSet.as_view({"get": "list", "head": "list"}),
+        name="form-list",
+    ),
     re_path(
         r"^enketo/(?P<xform_pk>\w+)/formList$",
         XFormListViewSet.as_view({"get": "list", "head": "list"}),
         name="form-list",
     ),
+    re_path(
+        r"^forms/(?P<xform_pk>\w+)/formList$",
+        XFormListViewSet.as_view({"get": "list", "head": "list"}),
+        name="form-list",
+    ),
```

Comment:
Could we also include `/submission$`? I expect that if `/projects/1/` is used the submission is expected to be at `/projects/1/submission` hence the path also should exist.

```python
re_path(
        r"^projects/(?P<xform_pk>\w+)/submission$",
        XFormSubmissionViewSet.as_view({"post": "create", "head": "create"}),
        name="submissions",
    ),
re_path(
        r"^forms/(?P<xform_pk>\w+)/submission$",
        XFormSubmissionViewSet.as_view({"post": "create", "head": "create"}),
        name="submissions",
    ),
```

Reply:
I wonder whether this should also extend to the ODK Briefcase API URLs with similar requirements.